test/safe/branch.test.ts

```diff
@@ -41,6 +41,137 @@ describe("Branch Webhook", () => {
 	describe("Create Branch", () => {
 		it("should queue and process a create webhook", async () => {
 
+			when(booleanFlag).calledWith(
```

Comment:
Copied the entire test describe block and had one with Octokit(to be cleaned/removed) and a ghclient flow.
We have 3 active Feature flags on this test now..... so will be nice to clean them up soon!

Reply:
Instead of copying the block, you can do `describe.each([true, false])("your description", (useNewGithubClient) => {...` since the code should work _exactly the same way_, it should pass whether it's on or off.  Here's an example: https://github.com/atlassian/github-for-jira/pull/1113/files#diff-d49caa3c6885aa6a494bf47acede68ba12da206329cb7eb0614b8fbd21d771e2R10